### PR TITLE
testmap: rhel-9-6 enablement

### PR DESCRIPTION
**Pull request by martinpitt**
First round: throw everything against the wall and see what sticks.

### CODE REVIEW

**Comment by martinpitt** [comment]
OK, so [vdo kmod](https://cockpit-logs.us-east-1.linodeobjects.com/pull-6919-58bbf6b2-20240926-093751-rhel-9-6-storage-cockpit-project-cockpit/log.html#104-2) is once again MIA, that's a recurring issue #4545. I'll re-add the naughty.

File: lib/testmap.py [hunk 1]
```diff
@@ -83,7 +83,6 @@ def contexts(image: str, *scenarios: Iterable[str], repo: str | None = None) ->
         ],
         '_manual': [
             'centos-9-bootc',
-            'rhel-9-5',
```
Comment (mvollmer) [inline lib/testmap.py:86]: Is this intentional? Everywhere else you remove 9-6.

Comment (martinpitt) [reply lib/testmap.py:86]: It is intentional, yes. We've traditionally not tested starter-kit on rhel, but I'd like to keep the latest rhel-9-Y in manual. I.e. read this as "move manual from 9-5 to 9-6". Sorry for the confusion.

Comment (mvollmer) [reply lib/testmap.py:86]: Alright!

**Comment by martinpitt** [comment]
Wait, [what](https://cockpit-logs.us-east-1.linodeobjects.com/pull-6919-a13b4956-20240926-100610-rhel-9-6-storage-cockpit-project-cockpit/log.html)? That's the vdo naughty I just re-added.. But the new testlib (webdriver-bidi) slightly changed the error message. I adjusted the naughties.

**Review by mvollmer — APPROVED** [review]
Thanks!

**Pull request merged**